chirps_utility.py

```diff
@@ -0,0 +1,140 @@
+from chirps import *
+from main import *
+from csv_utility import *
+from user_utility import *
+from random_id_generator import *
+
+class ChirpsUtility:
+
+	def view_chirps(public_convos, private_convos, chirps_dict, current_user):
+		"""
+		Prints out all public and private chirps that the current user is a part of
+
+		Arg- chirps dictionary (from CSV file), currently selected user
+		"""
+		# counter and empty list to be used for convo select
+		# (display counter as choice number, append each convo to convo_id_list,
+		# convo int(counter - 1) to find convo in convo_id_list by index
+		counter = 1
+		convo_id_list = list()
+		# if a user is signed in
+		if current_user != None:
+			try:
+				print("\n ~ PRIVATE CHIRPS ~ ")
+				for convo_key, chirps_list in private_convos.items():
+					# chirp info that started convo
+					convo_start = chirps_list[0]
+					# if current user is the recipient of a private chirp
+					if chirps_dict[convo_start][1] == current_user.screen_name:
+						print("{0}. {1}: {2}".format(counter, chirps_dict[convo_start][0], chirps_dict[convo_start][2]))
+						convo_id_list.append(convo_key, "private")
```

Comment:
Attempting to view chirps as a user who is the recipient of a private chirp crashes the program:
```
Welcome, rtanay!

What would you like to do?
1. New user account
2. Select user
3. View chirps
4. Public chirp
5. Private chirp
6. Quit
> 3

 ~ PRIVATE CHIRPS ~ 
1. bobo: Private wooooooo
Traceback (most recent call last):
  File "main.py", line 147, in <module>
    mm.show_menu()
  File "main.py", line 137, in show_menu
    self.show_menu()
  File "main.py", line 77, in show_menu
    self.current_user
  File "/Users/ryan/Desktop/birdyboard-josephkane/chirps_utility.py", line 30, in view_chirps
    convo_id_list.append(convo_key, "private")
TypeError: append() takes exactly one argument (2 given)
```
Looking at line 35 in this file, it seems it might just be a missing pair of square brackets.